### DISCUSSION TITLE
How to remove or reset `Favorite publications`?

### DISCUSSION THREAD

**Discussion by ashalfarhan** [dailydotdev/daily#485] [General]
Hi, it's nice that I can generate a dev card, but I'm confused about how the `Favourite publications` work, is it based on who was the publisher of what I've read or something? because I've never read about bootstrap anymore but I don't know why is there, is it possible to remove or just reset the favourite publications?

<img width="394" alt="Screenshot 2022-02-24 at 22 48 34" src="https://user-images.githubusercontent.com/70800415/155558532-092e6060-e93e-48eb-b5ef-3fc1b9ac6e9b.png">

**Comment by idoshamun** [comment]
Hi @ashalfarhan,

At this point, you can't choose the publications that are shown in the DevCard but you're right we have to update the query so it will not reflect publications that are not read anymore.
I'll let you know once we have some progress here.
